src/lib/radio/radio.ts

```diff
@@ -516,4 +498,20 @@ export class MdRadioButton implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
+  /** Function is called whenever the focus changes for the input element. */
+  private _onInputFocusChange(focusOrigin: FocusOrigin) {
```

Comment:
Are you sure about it? I felt like without `Change` it does not really feel like it would call for `blur` as well.